ngshare/ngshare.py

```diff
@@ -330,31 +354,51 @@ class ListCourses(MyRequestHandler):
     '/api/courses'
     @authenticated
     def get(self):
-        'List all available courses the user is taking or teaching (anyone)'
+        '''
+            List all available courses the user is taking or teaching. (anyone)
+            List all courses in ngshare. (admin)
+        '''
         courses = set()
-        for i in self.user.teaching:
-            courses.add(i.id)
-        for i in self.user.taking:
-            courses.add(i.id)
+        if self.is_admin():
+            for i in self.db.query(Course).all():
+                courses.add(i.id)
+        else:
+            for i in self.user.teaching:
+                courses.add(i.id)
+            for i in self.user.taking:
+                courses.add(i.id)
         self.json_success(courses=sorted(courses))
 
 class AddCourse(MyRequestHandler):
     '/api/course/<course_id>'
     @authenticated
     def post(self, course_id):
-        'Add a course (anyone)'
+        'Add a course (admin)'
+        self.check_admin()
         if self.db.query(Course).filter(Course.id == course_id).one_or_none():
             self.json_error(409, 'Course already exists')
         course = Course(course_id, self.user)
```

Comment:
Should we leave the course with no instructors, rather than adding the creator (admin user) as an instructor?

Reply:
Will allow user to pass in a list of instructors

Reply:
Fixed and pushed; also merged to `test-cov` branch (there is and is expected to be a conflict; fixed)